airbyte-config/config-models/src/main/resources/types/WellKnownTypes.yaml

```diff
@@ -0,0 +1,51 @@
+# Regexes are purely illustrative and need to be workshopped: BC dates shouldn't require 4-digit years; years may have >=5 digits; etc
+definitions:
+  String:
+    type: string
+    description: Arbitrary text
+  BinaryData:
```

Comment:
Should we prefix these with Airbyte, e.g. `AirbyteBinaryData` to be clear that these are are own types (vs JSON-schema)

Reply:
@cgardens I think you had some thoughts around using the `airbyte` prefix?

IMO being namespaced under WellKnownTypes.yaml (or whatever we end up calling it) should be enough to communicate that these aren't the builtin json schema types, especially since the usage is different (`type: string` vs `$ref:....String`). I could be convinced otherwise though.